src/components/Home/BookList/BookList.js

```diff
@@ -1,17 +1,25 @@
+import { useEffect } from 'react';
 import './BookList.css';
-import { useSelector } from 'react-redux';
+// import axios from 'axios';
```

Comment:
- [ ] For clean code and best practices, kindly make sure you remove all commented and unused code lines from your code base